R/later.R

```diff
@@ -269,6 +269,76 @@ later <- function(func, delay = 0, loop = current_loop()) {
   invisible(create_canceller(id, loop$id))
 }
 
+#' Executes a function when a file descriptor is ready
+#'
+#' Schedule an R function or formula to run after an indeterminate amount of
+#' time when file descriptors are ready for reading or writing, subject to an
+#' optional timeout.
```

Comment:
Maybe add a note that if poll/select itself errors, you'll get called back with a  `NA`? (And acknowledge that this is not distinguishable from the case where poll/select succeeded but had error conditions pending on each fd)

Reply:
Added a short note in [d8eb8b7](https://github.com/r-lib/later/pull/190/commits/d8eb8b723dd374460276e27d4c15b2a55d2a1cf7).